int64 timestamp
int32 steer